wormhole-connect/src/views/v2/Redeem/index.tsx

```diff
@@ -68,6 +68,15 @@ const useStyles = makeStyles()((_theme) => ({
     maxWidth: '420px',
     width: '100%',
   },
+  errorBox: {
+    maxWidth: '420px',
+  },
+  txStatusIcon: {
+    width: 105,
+    height: 105,
+    marginTop: '1rem',
```

Comment:
We haven't been using rem for spacing, this may cause an issue when expanding/shrinking the window I believe.

Reply:
Changed everything to px.